app/_data/kong_versions.yml

```diff
@@ -133,6 +133,16 @@
     cassandra: "3.x.x"
     postgres: "9.5+"
     openresty: "1.13.6.2"
+-
+  release: "1.0.x"
+  version: "1.0.0"
+  luarocks_version: "1.0.0-0"
+  dependencies:
+    luajit: "2.1.0-beta2"
```

Comment:
This should be `2.1.0-beta3` (I think it has been long like that, we just have forgotten it), you can correct it on `0.14.x` too.